packages/theming/theme-tokens/src/getShadowTokens.ts

```diff
@@ -0,0 +1,15 @@
+import lightShadowTokens from '@fluentui-react-native/design-tokens-windows/light/tokens-shadow.json';
+import darkShadowTokens from '@fluentui-react-native/design-tokens-windows/dark/tokens-shadow.json';
+import hcShadowTokens from '@fluentui-react-native/design-tokens-win32/hc/tokens-shadow.json';
+import { AppearanceOptions } from '@fluentui-react-native/theme-types';
+
+export function getShadowTokens(mode: AppearanceOptions) {
+  if (mode === 'light') {
+    return lightShadowTokens.shadow;
+  } else if (mode === 'dark') {
+    return darkShadowTokens.shadow;
+  }
+
+  // HC mode.
+  return hcShadowTokens.shadow;
```

Comment:
Shadows are fundamentally different on Android, so I'd expect this file to change in the future. I guess it's ok as is.